src/main/java/org/example/jakarta_labb/validate/CustomReleaseYearValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.example.jakarta_labb.validate;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class CustomReleaseYearValidator implements ConstraintValidator<ReleaseYear, LocalDate> {
+
+    @Override
+    public void initialize(ReleaseYear constraintAnnotation) {
+        ConstraintValidator.super.initialize(constraintAnnotation);
+    }
+
+    @Override
+    public boolean isValid(LocalDate releaseYear, ConstraintValidatorContext context) {
```

Comment:
Should this be an int? In the MovieDto the type is int.

Reply:
Yes you are correct, I was playing around with the LocalDate to try to make it more dynamic. I corrected it now! 😄